tests/unit/test_dbapi_connection.py

```diff
@@ -27,19 +32,32 @@ def _get_target_class():
     def _make_one(self, *args, **kw):
         return self._get_target_class()(*args, **kw)
 
-    def _mock_client(self, rows=None, schema=None):
+    def _mock_client(self):
         from google.cloud.bigquery import client
 
         mock_client = mock.create_autospec(client.Client)
         return mock_client
 
+    def _mock_bqstorage_client(self):
+        from google.cloud.bigquery_storage_v1beta1 import client
+
+        mock_client = mock.create_autospec(client.BigQueryStorageClient)
+        return mock_client
+
+    @unittest.skipIf(
+        bigquery_storage_v1beta1 is None, "Requires `google-cloud-bigquery-storage`"
+    )
     def test_ctor(self):
         from google.cloud.bigquery.dbapi import Connection
 
         mock_client = self._mock_client()
-        connection = self._make_one(client=mock_client)
+        mock_bqstorage_client = self._mock_bqstorage_client()
+        connection = self._make_one(
+            client=mock_client, bqstorage_client=mock_bqstorage_client,
+        )
         self.assertIsInstance(connection, Connection)
         self.assertIs(connection._client, mock_client)
+        self.assertIs(connection._bqstorage_client, mock_bqstorage_client)
```

Comment:
Seems like we need an explicit test for the constructor without the `bqstorage_client` argument passed?